src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,4 +1,83 @@
 databaseChangeLog:
   -  changeSet:
        id:  adicionar suporte ao liquibase
-       author:  izumizawa
\ No newline at end of file
+       author:  izumizawa
+
+  -  changeSet:
+       id:  Adding telephone type table
+       author:  izumizawa
+       changes:
+         -  createTable:
+              tableName: telephone_type
+              columns:
+                - column:
+                    name: telephone_id_type
+                    type: int
+                    constraints:
+                      nullable: false
+                      primaryKey: true
+                - column:
+                    name: telephone_type
+                    type: varchar(20)
+
+  -  changeSet:
+       id:  Adding telephone table
+       author:  izumizawa
+       changes:
+         -  createTable:
+              tableName: telephone
+              columns:
+                -  column:
+                     name: telephone_id
+                     type: uuid
+                     constraints:
+                       primaryKey: true
+                       nullable: false
+                -  column:
+                     name: client_id
+                     type: uuid
+                     constraints:
+                       nullable: false
+                -  column:
+                     name: telephone_id_type
+                     type: int
+                     constraints:
+                       foreignKeyName: telephone_id_type
```

Comment:
maybe add "_fk" to FK name so we know this constraint is a FK, like "telefone_type_fk"